ipatests/test_integration/test_idm_api.py

```diff
@@ -8,6 +8,7 @@
 
 API_INIT = """
     from ipalib import api, errors
+    from pprint import pprint as pp
```

Comment:
Please remove the import, `pp` is not used in the test.

Reply:
Fixed.